src/components/rogerparada/PasswordGenerator.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+	<div class="w-96 h-56 border-white border-2 bg-white">
+		<p class="text-3xl text-center mt-3">Password generator</p>
+		<div class="row flex justify-center gap-0 mt-5 w-80 mx-auto">
+			<input
+				type="text"
+				name="passwordBox"
+				id="passwordBox"
+				readonly
+				class="p-1 border-2 border-gray-400 w-72"
+				v-model="password"
+			/>
+			<button
+				class="border-2 p-1 border-gray-400 bg-white hover:bg-black border-l-0"
+				@click="copyPassword()"
+			>
+				<img alt="Copy" title="Copy" :src="iconCopy" class="w-8 h-8" />
+			</button>
+		</div>
+		<div class="flex justify-center">
+			<input type="range" name="" id="" v-model="passLength" class="w-80 mt-3" />
+		</div>
+		<div class="flex justify-center mt-3 text-white">
+			<button @click="generatePassword()" class="bg-gray-500 p-3">Generate Password</button>
+		</div>
+		<div class="slider"></div>
+	</div>
+</template>
+
+<script>
+import { ref } from '@vue/reactivity'
+import icon from './assets/images/iconCopy.svg'
```

Comment:
Hola!! 👋 El error del build es porque no se ha podido encontrar este directorio! Te dejo la solución aquí abajo! 👇
```suggestion
import icon from './assets/images/IconCopy.svg'
```

Reply:
Gracias, corregido y subido.